### PR TITLE
(#910)(#882) Fix URL's and make consistent

### PR DESCRIPTION
**Pull request by pauby**
## Description Of Changes
Fixes invalid URL's and changes text to make it consistent with other docs.

## Motivation and Context
It's broken.

## Testing

* [x] I have previewed these changes using the [Docker Container](https://github.com/chocolatey/docs/tree/master/.devcontainer) or another method before submitting this pull request.

## Change Types Made

* [x] Minor documentation fix (typos etc.).
* [ ] Major documentation change (refactoring, reformatting or adding documentation to existing page).
* [ ] New documentation page added.
* [ ] The change I have made should have a video added, and I have raised an issue for this.
    * Issue #

## Change Checklist

* [ ] Requires a change to menu structure (top or left-hand side)/
* [ ] Menu structure has been updated

## Related Issue
#910 
#882

### CODE REVIEW

File: input/en-us/guides/create/create-msi-package.md [hunk 1]
```diff
@@ -177,10 +177,10 @@ In the case of the example package, Chocolatey handles it well - so you can remo
 
 You can now run `choco pack` to compile your Chocolatey package, creating a file with a `.nupkg` extension, ready for installation!
 
-1. In VSCode, press `Ctrl + Shift + P` or use the **View** menu and click on **Command Palette**.
+1. In VSCode, press **Ctrl+Shift+P** or use the **View** menu and click on **Command Palette**.
 1. Select `Chocolatey: Package Chocolatey package(s)` from the prompt.
 1. Select `firefox-msi.nuspec` from the prompt.
-1. In the `additional arguments` dialog enter `--output-directory='~\tutorials'` (or whichever directory you're using for these tutorials), and press Enter.
+1. In `Additional Arguments` enter `--output-directory='~\tutorials'` (or whichever directory you're using for these tutorials), and press **Enter**.
```
Comment (AdmiringWorm) [inline input/en-us/guides/create/create-msi-package.md:183]: IMO, this should maybe be

```suggestion
1. In **Additional Arguments** enter `--output-directory='~\tutorials'` (or whichever directory you're using for these tutorials), and press `Enter`.
```

File: input/en-us/guides/create/create-msi-package.md [hunk 2]
```diff
@@ -177,10 +177,10 @@ In the case of the example package, Chocolatey handles it well - so you can remo
 
 You can now run `choco pack` to compile your Chocolatey package, creating a file with a `.nupkg` extension, ready for installation!
 
-1. In VSCode, press `Ctrl + Shift + P` or use the **View** menu and click on **Command Palette**.
+1. In VSCode, press **Ctrl+Shift+P** or use the **View** menu and click on **Command Palette**.
```
Comment (AdmiringWorm) [inline input/en-us/guides/create/create-msi-package.md:180]: Same as above

```suggestion
1. In VSCode, press `Ctrl+Shift+P` or use the **View** menu and click on **Command Palette**.
```

Or

```suggestion
1. In VSCode, press <kbd>Ctrl</kbd> + <kbd>Shift</kbd> + <kbd>P</kbd> or use the **View** menu and click on **Command Palette**.
```

File: input/en-us/guides/create/create-zip-package.md [hunk 3]
```diff
@@ -21,7 +21,7 @@ that you include the Zip archive as part of the package. We'll look at downloadi
 Perform the following steps:
 
 1. Open the `tutorials` folder in VSCode.
-2. Press `Ctrl+Shift+P` or select **View** > **Command Palette**.
+2. Press **Ctrl+Shift+P** or select **View** > **Command Palette**.
```
Comment (AdmiringWorm) [inline input/en-us/guides/create/create-zip-package.md:24]: Similar to the suggestions above regarding command keys

```suggestion
1. Press `Ctrl+Shift+P` or select **View** > **Command Palette**.
```

Or

```suggestion
1. Press <kbd>Ctrl</kbd> + <kbd>Shift</kbd> + <kbd>P</kbd> or select **View** > **Command Palette**.
```

File: input/en-us/guides/create/create-zip-package.md [hunk 4]
```diff
@@ -113,10 +112,10 @@ In the VSCode Explorer pane, find and open the `embedded-zip.nuspec` file. Repla
 
 The `pack` command is used to compile your Chocolatey package, giving it a `.nupkg` extension.
 
-1. In VSCode press `Ctrl+Shift+P` or **View** > **Command Pallete**.
+1. In VSCode press **Ctrl+Shift+P** or use the **View** menu and click on **Command Palette**.
```
Comment (AdmiringWorm) [inline input/en-us/guides/create/create-zip-package.md:115]: Similiar to the suggestions made above regarding the command keys

```suggestion
1. In VSCode, press `Ctrl+Shift+P` or use the **View** menu and click on **Command Palette**.
```

Or

```suggestion
1. In VSCode, press <kbd>Ctrl</kbd> + <kbd>Shift</kbd> + <kbd>P</kbd> or use the **View** menu and click on **Command Palette**.
```

File: input/en-us/guides/create/create-zip-package.md [hunk 5]
```diff
@@ -113,10 +112,10 @@ In the VSCode Explorer pane, find and open the `embedded-zip.nuspec` file. Repla
...
 2. Type `Chocolatey:` and click `Chocolatey: Pack Chocolatey package(s)`.
 3. Select `embedded-zip.nuspec` from the list.
-4. In the `additional arguments` dialog enter `--output-directory='C:\tutorials'` and press `Enter`.
+4. In `Additional arguments` enter `--output-directory='C:\tutorials'` and press **Enter**.
```
Comment (AdmiringWorm) [inline input/en-us/guides/create/create-zip-package.md:118]: IMO, this should maybe be

```suggestion
1. In **Additional Arguments** enter `--output-directory='C:\tutorials'` and press `Enter`.
```

File: input/en-us/guides/create/create-msi-package.md (hunk 1 above)
Comment (pauby) [reply input/en-us/guides/create/create-msi-package.md:183]: This isn't consistent with how it is in other documents. This PR is about adding that consistency.

File: input/en-us/guides/create/create-msi-package.md [hunk 6]
```diff
@@ -43,12 +43,12 @@ In this example, we're going to be creating a new Firefox MSI package.
...
 > In this case, we browsed to [Mozilla's download site](https://www.mozilla.org/en-GB/firefox/all/#product-desktop-release) and grabbed the URL of the MSI(s) we wanted.
 
-There are [package templates](#xref:create-custom-package-templates) to create various package types, including an MSI template!
+There are [package templates](xref:create-custom-package-templates) to create various package types, including an MSI template!
 
 We'll use this to create our new MSI package:
 
 1. Open your `tutorials` directory in VSCode.
-1. Press `Ctrl + Shift + P` (or select **View** > **Command Palette**).
+1. Press **Ctrl+Shift+P** (or select **View** > **Command Palette**).
```
[the remark this answers is not in the document]
Comment (pauby) [reply input/en-us/guides/create/create-msi-package.md:51]: This is also about it being consistent with other documents.

**Comment by pauby** [comment]
This has been updated @AdmiringWorm

**Review by AdmiringWorm — APPROVED** [review]
LGTM

**Pull request merged**